### ISSUE TITLE
Change bindings at runtime / interop?

### ISSUE THREAD

**Issue by bovine3dom** [mooz/xkeysnail#34]
Hi,

I help make [Tridactyl](https://github.com/tridactyl/tridactyl) and I recommend xkeysnail to our users for working around the fact that we can't rebind some keys (e.g, Ctrl-N).

I'd like to be able to use xkeysnail to rebind keys per-mode in Tridactyl, so that I can bind e.g, `d` to Ctrl-W in Tridactyl's normal mode and then disable that bind in other modes, via e.g.

`xkeysnail-remote --mode="normal"` <- `d` bind activated

`xkeysnail-remote --mode="insert"` <- `d` bind deactivated

Is this something you'd be interested in adding? (Relevant-ish Tridactyl issue: https://github.com/tridactyl/tridactyl/issues/359)

I'm keen to avoid people having to give sudo permissions to Tridactyl and I want to keep latency to a minimum (typing in a textbox and having the tab close would be lame), so I suspect `killall xkeysnail; xkeysnail ignore.py` wouldn't be good enough.

Thanks.

**Comment by joshgoebel** [comment]
> I'd like to be able to use xkeysnail to rebind keys per-mode in Tridactyl,

Could you do this with a conditional keymap if we supported WM_NAME just by changing the name of your window slightly?

**Comment by bovine3dom** [comment]
Yeah, that should work fine, we can add a prefix to the window quite easily : )
